src/components/IconButton.tsx

```diff
@@ -9,7 +9,7 @@ type IconButton = {
 export function IconButton(props: IconButton) {
   return (
     <Link href="/account">
-      <CgProfile className="w-6 h-6 text-primary-default-Solid" />
+      <CgProfile className="w-12 h-12 text-primary-default-Solid" />
```

Comment:
you could pass in a prop to this IconButton component that allows a user of the component to add classes

Reply:
ha sorry, just realized this was pending for a week. i thought it was done and picked others issues in between. Im going to look at what you say.